Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/Bookmarks/AddEditBookmarkTableViewController.swift

```diff
@@ -235,9 +235,21 @@ class AddEditBookmarkTableViewController: UITableViewController {
         // Saving and udpating is slightly different for each mode.
         switch mode {
         case .addBookmark(_, _):
-            guard let urlString = bookmarkDetailsView.urlTextField?.text,
-                let url = URL(string: urlString) else {
+            guard let urlString = bookmarkDetailsView.urlTextField?.text else {
+                return earlyReturn()
+            }
+            
+            var url: URL! = URL(string: urlString)
```

Comment:
unwrap it, add it to the guard statement few lines above, should work with `var`